auxiliary.py

```diff
@@ -181,6 +153,46 @@ def plot_axial(data, vb, vc, vt):
     plt.savefig('axial1', dpi=300, bbox_inches="tight")
 
 
+def plot_detector(data, name, V=1):
+    """
+    Plots flux in the axial direction 'Z'.
+
+    Parameters:
+    -----------
+    data: [serpenttools format]
+    name: [string]
+    name of the detector
+    V: [float]
+    total volume where the detector is applied [cm3]
+    """
+
+    det = data.detectors[name]
+    z = [line[0] for line in det.grids['Z']]
+    val = det.tallies
+    vdetector = V/len(z)
+    val = val/vdetector
+
+    plt.figure()
+    plt.step(z, val[0], where='post', label='thermal')
+    plt.step(z, val[1], where='post', label='fast')
+    plt.xlabel('z [cm]')
+    plt.ylabel(r'$\phi$')
+    plt.legend(loc="upper right")
+    plt.savefig(name, dpi=300, bbox_inches="tight")
+
+
+def plots_standardcolum():
```

Comment:
Is this a typo? Do you mean plots_standardcolumn()?

Reply:
you are certainly right